vignettes/GOstatsForUnsupportedOrganisms.Rmd

```diff
@@ -18,33 +18,34 @@ output:
 # Introduction
 
 This vignette is meant as an extension of what already exists in the
-GOstatsHyperG.pdf vignette. It is intended to explain how a user can run
-hypergeometric testing on GO terms or KEGG terms when the organism in
-question is not supported by an annotation package. The 1st thing for a
-user to determine then is whether or not their organism is supported by
-an organism package through AnnotationForge. In order to do that, they
-need only to call the available.dbschemas() method from AnnotationForge.
+[GOstatsHyperG.html](https://bioconductor.org/packages/devel/bioc/vignettes/GOstats/inst/doc/GOstatsHyperG.html)
+vignette. It is intended to explain how a user can run hypergeometric testing
+on GO terms or KEGG terms when the organism in question is not supported by
+an annotation package. The 1st thing for a user to determine then is whether
+or not their organism is supported by an organism package through
+`r Biocpkg("AnnotationForge")`. In order to do that, they need only to call the
+available.dbschemas() method from `r Biocpkg("AnnotationForge")`.
 
 ```{r available Schemas, message=FALSE, warning=FALSE}
 library("AnnotationForge")
 available.dbschemas()
```

Comment:
Here we find some differences in the output of available.dbschemas() function  like for example we have "WORM_DB" near [31] in original pdf but we have "WORM_DB"  near [28] in the html file I don't think this is any problem but pointing out as a difference from original pdf